### DISCUSSION TITLE
Repeated Chat Message Detection

### DISCUSSION THREAD

**Discussion by loaymohamad** [Chatterino/chatterino2#4517] [Ideas]
Show in chat how often a user repeated a message like Chatty does

**Comment by pajlada** [comment]
Could you explain a bit more in detail what you mean with this idea?

**Reply by loaymohamad** [reply]
A feature that uses the user's message history to determine if the user has sent the same (or similar) message before. If a repeated message has been detected (depending on the settings) the number of repeated messages is added behind the message in chat, "(x4)". So Moderators can see who repeats a message how often times and timeout them.
